query/src/main/scala/filodb/query/exec/PeriodicSamplesMapper.scala

```diff
@@ -24,7 +24,7 @@ final case class PeriodicSamplesMapper(start: Long,
                                        end: Long,
                                        window: Option[Long],
                                        functionId: Option[RangeFunctionId],
-                                       funcParams: Seq[Any] = Nil) extends RangeVectorTransformer {
+                                       funcParams: Seq[Any] = Nil) extends RangeVectorTransformer with StrictLogging {
```

Comment:
Should not add logger here - it will affect serialization. Use query logger instead.